tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/validation/TclMissingFixtureValidatorTest.xtend

```diff
@@ -65,6 +67,27 @@ class TclMissingFixtureValidatorTest extends AbstractMockedTclValidatorTest {
 		messageAcceptor.verify(never).acceptInfo(anyString, anyObject, anyObject, anyInt, anyString)
 	}
 
+	@Test
+	def void justMissingExceptionInfoOnExistingFixture() {
+		// given
+		val tclFix = parseTcl('''
+			package pa
+			# Test
+			
+			* first
+			Component: some_fantasy_component
+			- test step that maps
+		''')
+		val testStepThatMaps = tclFix.test.steps.head.contexts.head.assertInstanceOf(ComponentTestStepContext).steps.head.assertInstanceOf(TestStep)
+
+		// when
+		tclValidator.checkFixtureMethodForExistence(testStepThatMaps)
+
+		// then
+		messageAcceptor.verify.acceptInfo(message.capture, anyObject, anyObject, anyInt, anyString)
+		assertMatches(message.value, ".*does not provide additional information on failures.*")
```

Comment:
… was about to remark that the message should probably name the exception that the fixture method should declare (FixtureException) – but it actually does. So shouldn't the test assert that, as well (instead of using ".*" regex matching)?